docs/getting-started-guides/ubuntu/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Kubernetes on Ubuntu
+redirect_from: /docs/getting-started-guides/ubuntu-calico/
```

Comment:
It looks like the line is [already in the file](https://github.com/kubernetes/kubernetes.github.io/blob/master/_redirects#L84).  👍  I'll just remove the `redirect_from` statement.